romancal/source_catalog/psf.py

```diff
@@ -69,20 +69,20 @@ def create_gridded_psf_model(
         Computed gridded PSF model for this SCA.
         Examples include: `"SCA01"` or `"SCA18"`.
     oversample : int, optional
-        Oversample factor, default is 11. See WebbPSF docs for details [1]_.
+        Oversample factor, default is 11. See Stpsf docs for details [1]_.
         Choosing an odd number makes the pixel convolution more accurate.
     fov_pixels : int, optional
         Field of view width [pixels]. Default is 12.
-        See WebbPSF docs for details [1]_.
+        See Stpsf docs for details [1]_.
     sqrt_n_psfs : int, optional
         Square root of the number of PSFs to calculate, distributed uniformly
         across the detector. Default is 4.
     buffer_pixels : int, optional
         Calculate a grid of PSFs distributed uniformly across the detector
         at least ``buffer_pixels`` away from the detector edges. Default is 100.
     instrument_options : dict, optional
-        Instrument configuration options passed to WebbPSF.
-        For example, WebbPSF assumes Roman pointing jitter consistent with
+        Instrument configuration options passed to Stpsf.
```

Comment:
```suggestion
        Instrument configuration options passed to STPSF.
```
I believe the name is `STPSF` based on the wording on https://stpsf.readthedocs.io/en/latest/webbpsf_to_stpsf.html#Transitioning-from-WebbPSF-to-STPSF